test/integration/targets/play_iterator/runme.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -ux
```

Comment:
Although not necessary for the current test, this would be good to include in case someone extends the test later and doesn't notice that `-e` is missing, since it's commonly included in all of our script based tests.

```suggestion
set -eux
```

Reply:
Done.